ddtrace/tracer/option_test.go

```diff
@@ -324,7 +324,37 @@ func TestTracerOptionsDefaults(t *testing.T) {
 		defer os.Unsetenv("DD_AGENT_HOST")
 		tracer := newTracer()
 		c := tracer.config
-		assert.Equal(t, "trace-agent:8126", c.agentAddr)
+		assert.Equal(t, "http://trace-agent:8126", c.agentURL)
+	})
+
+	t.Run("env-agentURL", func(t *testing.T) {
+		t.Run("env", func(t *testing.T) {
+			os.Setenv("DD_TRACE_AGENT_URL", "https://custom:1234")
```

Comment:
t.SetEnv will simplify these tests greatly

Reply:
Changed this to `t.Setenv` 👍